shotover-proxy/tests/test-configs/kafka/passthrough-sasl/docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: '2'
+
+services:
+  kafka:
+    image: 'bitnami/kafka:latest'
```

Comment:
Please use a specific release instead of latest.
I recommend `bitnami/kafka:3.6.1-debian-11-r24` to match up with https://github.com/shotover/shotover-proxy/pull/1487